cfpb/cfgov-refresh
# Adding second division lead to Bureau structure page

Adding second division lead to Bureau structure page

## Changes

- Modified code to add second division lead to bureau structure chart. 

## Testing

1.  Visit the bureau structure page in the Wagtail admin.
2. Edit the `Division Lead:` input field.
3.  Edit the `Title 1:` input field.
4.  Edit the `Title 2:` input field.
5. Publish the chart and verify the second lead appears on the chart. 

## Screenshots

<img width="1042" alt="screen shot 2018-04-25 at 3 49 50 pm" src="https://user-images.githubusercontent.com/1696212/39269247-5d7e590c-48a0-11e8-8ac5-d50e19b84842.png">
<img width="443" alt="screen shot 2018-04-25 at 4 00 55 pm" src="https://user-images.githubusercontent.com/1696212/39269809-e3a2bc70-48a1-11e8-8f1b-bc9d2f81d8c2.png">



## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
